decidim-core/app/views/decidim/pages/home.html.erb

```diff
@@ -1,17 +1,3 @@
-<%= render partial: "decidim/pages/home/hero" %>
-
-<% if !translated_attribute(current_organization.description).blank? %>
-  <%= render partial: "decidim/pages/home/sub_hero" %>
+<% Decidim::ContentBlock.where(organization: current_organization, scope: :homepage).where.not(published_at: nil).order(weight: :asc).each do |content_block| %>
```

Comment:
Maybe move all these conditions to a scope? (Also, related to the index comment for the database)